.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
- [ ] Kindly update your README file with the `microverse` template instead of the `react` default README to make the repo more professional. you can find it [here](https://github.com/microverseinc/readme-template)